attributes/default.rb

```diff
@@ -15,3 +15,7 @@
 
 # PGP Key Server
 default['le']['pgp_key_server'] = 'pgp.mit.edu'
+
+# Debian Release
+
+default['le']['deb'] = 'trusty'
```

Comment:
Sorry for the delay in getting back to you, I'd set the attribute to default to the following:

```ruby 
default['le']['deb'] =  node['lsb']['codename']
```

That way users can override if they wish and this doesnt change the current behavior.